### PR TITLE
Add initial snackbar docs

### PR DESCRIPTION
**Pull request by guidezpl**
* Modified the template so that **Anatomy and key properties** is 3rd-level heading instead of 2nd, like other docs
* Added `Elevation` and `Shape border` to **Container properties**

Question: Should the docs link to https://flutter.dev/docs/cookbook/design/snackbars?

TODO: in both DartPad demos, replace link for snackbar-background.jpg after merging

### CODE REVIEW

File: docs/components/dartpad/snackbars/regular/main.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    );
+  }
+}
+
+class SnackbarButton extends StatelessWidget {
+  Widget build(BuildContext context) {
+    return RaisedButton(
+      onPressed: () {
+        final snackBar = SnackBar(
+          content: Text('Text label'),
```
Comment (rami-a) [inline docs/components/dartpad/snackbars/regular/main.dart:41]: Can you set behavior to floating. We should try to make that the new default

File: docs/components/dartpad/snackbars/regular/main.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        );
+
+        // Find the Scaffold in the widget tree and use
+        // it to show a SnackBar.
+        Scaffold.of(context).showSnackBar(snackBar);
+      },
+      child: Text('Show SnackBar'),
+    );
+  }
+}
```
Comment (rami-a) [inline docs/components/dartpad/snackbars/regular/main.dart:55]: Add new line at end of file

File: docs/components/dartpad/snackbars/theme/main.dart [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    );
+  }
+}
+
+class SnackbarButton extends StatelessWidget {
+  Widget build(BuildContext context) {
+    return RaisedButton(
+      onPressed: () {
+        final snackBar = SnackBar(
+          content: Text('Text label'),
```
Comment (rami-a) [inline docs/components/dartpad/snackbars/theme/main.dart:42]: Can you set behavior to floating here

File: docs/components/snackbars.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+| **Text color** | pass a Widget to `content`|
+| **Typography** | pass a Widget to `content`|
+
+#### Container properties
+
+&nbsp; | Property
+------ | --------- 
+**Color** | `backgroundColor`
+**Elevation** | `elevation`
+**Height** |  N/A
```
Comment (rami-a) [inline docs/components/snackbars.md:55]: I think the recommendation is to remove N/A properties

File: docs/components/snackbars.md [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+* [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/snack_bar.dart)
+* [Dartpad Demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/snackbars/regular)
+
+The following is an example of a snackbar with an action button:
+
+![Snackbar example](assets/snackbars/snackbars-regular.png)
+
+```dart
+SnackBar(
+  content: Text('Text label'),
```
Comment (rami-a) [inline docs/components/snackbars.md:81]: Add floating behavior

File: docs/components/snackbars.md [hunk 6]
```diff
@@ -0,0 +1,238 @@
...
+    );
+  }
+}
+
+class SnackbarButton extends StatelessWidget {
+  Widget build(BuildContext context) {
+    return RaisedButton(
+      onPressed: () {
+        final snackBar = SnackBar(
+          content: Text('Text label'),
```
Comment (rami-a) [inline docs/components/snackbars.md:143]: Add floating behavior

File: docs/components/snackbars.md (hunk 4 above)
Comment (guidezpl) [reply docs/components/snackbars.md:55]: done

**Pull request merged**